lib/Sema/CSBindings.cpp

```diff
@@ -374,6 +375,10 @@ ConstraintSystem::getPotentialBindings(TypeVariableType *typeVar) {
         result.InvolvesTypeVariables = true;
       continue;
     }
+    
+    // Do not attempt to bind to ErrorType.
+    if (type->hasError())
```

Comment:
`compiler_crashers_fixed/25750-swift-lvaluetype-get.swift` forms a constraint that tries to form a value member constraint directly to an `ErrorType`.